lib/r_enum/enumerable/ruby.ex

```diff
@@ -644,11 +644,7 @@ defmodule REnum.Enumerable.Ruby do
   """
   def tally(enumerable) do
```

Comment:
Since the implementation are the same, it seems good to delegate it like other function aliases!
And deletes docs too.
`defdelegate tally(enumerable), to: Enum, as: :frequencies`

Reply:
Thanks for your review.

- add: `defdelegate tally(enumerable), to: Enum, as: :frequencies`
- delete: docs 

I force push.
Would you please review again?

Reply:
@TORIFUKUKaiou 
Thanks！Looks good to me！
Conflicts are fierce, so I'll resolve them and merge them!

Reply:
@TORIFUKUKaiou 
Sorry. I can't push your PR.Can you resolve conflicts?
Renamed `lib/r_enum/enumerable/ruby.ex` to `lib/r_enum/ruby.ex`.